Automattic/jetpack
# Decouple action bar components

## Proposed changes:
This PR takes the action bar components out of the inbox component. They lack better styling and functionality. 

### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Enable package and new inbox page:
```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

Navigate to /wp-admin/admin.php?page=jetpack-forms

See the actions bar resembles the design. 
Search is functional via "Enter" key.
Bulk actions is not functional, though the dropdown should work fine
